video/out/opengl/context_angle.c

```diff
@@ -203,14 +239,28 @@ static int angle_init(struct MPGLContext *ctx, int flags)
         goto fail;
     }
 
-    EGLint d3d_types[] = {EGL_PLATFORM_ANGLE_TYPE_D3D11_ANGLE,
-                          EGL_PLATFORM_ANGLE_TYPE_D3D9_ANGLE};
+    struct {
+        EGLint platform, device_type;
+    } d3d_types[] = {
```

Comment:
(Though that would make it harder to do the device_type check at the end, which is necessary for showing a warning message on Windows 7.)